src/Gui/Gui.cpp

```diff
@@ -209,6 +209,12 @@ bool Gui::ProcessEventSdl2(const SDL_Event& event)
 		io.KeySuper = ((SDL_GetModState() & KMOD_GUI) != 0);
 		return io.WantCaptureKeyboard;
 	}
+	case SDL_WINDOWEVENT:
+		if (event.window.event == SDL_WINDOWEVENT_SIZE_CHANGED)
+		{
+			bgfx::reset(static_cast<uint32_t>(event.window.data1), static_cast<uint32_t>(event.window.data2), BGFX_RESET_VSYNC);
```

Comment:
Why does this have to be caught in the Gui? This is a bad place to have this code.

Reply:
I've tested resizing without this and it doesn't contribute.

Reply:
In the event loop (https://github.com/openblack/openblack/blob/ddf190ae27a2779adb260c2719b87d09f8fbb0ce/src/Game.cpp#L131) the Gui is the first to handle SDL events.

By default, if an event is out of the window it is dropping it (https://github.com/openblack/openblack/blob/ddf190ae27a2779adb260c2719b87d09f8fbb0ce/src/Gui/Gui.cpp#L215)
The problem in this case is that the window cannot be scaled up.

So an important part in the event handling in Gui.cpp is the `return false`.

However, as you said, the bgfx::reset is useless. It's already present in the Game::ProcessEvents function. I'm removing it from the Gui.

Reply:
All that remains to do is store the reset flags in the renderer and reuse them during reset.

Reply:
I added a uint32_t _bgfxReset member in Renderer for saving the reset flags. This should solve the last issue we had.